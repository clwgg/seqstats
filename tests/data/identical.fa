>seq1
ACGT
>seq2
TGCA
>seq3
GATC
>seq4
CTAG
