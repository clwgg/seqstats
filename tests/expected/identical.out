Total n:	4
Total seq:	16 bp
Avg. seq:	4.00 bp
Median seq:	4.00 bp
N 50:		4 bp
Min seq:	4 bp
Max seq:	4 bp
